README.md

```diff
@@ -813,6 +813,7 @@ Clients for commercial social platforms that had their API access cut off in a w
 
 - [Gnome Commander](https://gitlab.gnome.org/GNOME/gnome-commander) - Fast and powerful twin-panel file manager `#c++` `#rust` `#gtk3`.
 - [Hyperplane](https://github.com/kra-mo/hyperplane) - Non-hierarchical file manager `#python` `#gtk4` `#libadwaita`.
+- [Gnome Files (Nautilus)](https://gitlab.gnome.org/GNOME/nautilus) - Default file manager of the GNOME desktop `#c` `#gtk4` `#libadwaita` `#gnome`. 
```

Comment:
```suggestion
- [GNOME Files (Nautilus)](https://apps.gnome.org/Nautilus) - Default file manager of the GNOME desktop `#c` `#gtk4` `#libadwaita` `#gnome`. 
```

Funny that the good old Nautilus wasn't on the page yet! I would use the `apps.gnome.org` page instead since it has screenshots.